README.md

```diff
@@ -2,6 +2,8 @@
 
 [![Build Status](https://travis-ci.org/stuartgunter/dropwizard-logging-config.png?branch=master)](https://travis-ci.org/stuartgunter/dropwizard-logging-config)
 
+**Note: This task is now included in the Dropwizard core and is enabled by default for the admin environment.**
```

Comment:
Thanks for adding the note. Please could you include the version of Dropwizard that includes this task so anyone looking at it knows when the library is needed or not?